jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/AbstractHttpTest.java

```diff
@@ -57,8 +60,16 @@ public void setUp() throws Exception
     @AfterEach
     public void tearDown() throws Exception
     {
-        server.stop();
-        stacklessChannelLogging.close();
+        try
+        {
+            Set<ArrayByteBufferPool.Tracking.Buffer> serverLeaks = bufferPool.getLeaks();
+            assertEquals(0, serverLeaks.size(), bufferPool.dumpLeaks());
+        }
+        finally
+        {
+            server.stop();
+            stacklessChannelLogging.close();
+        }
```

Comment:
Can you re-arrange to avoid the `finally`...
getLeaks()
stop()
close()
assert()